Dockerfile

```diff
@@ -1,5 +1,27 @@
-FROM alpine
-COPY bin/helpwave /usr/local/bin/helpwave
-RUN chmod +x /usr/local/bin/helpwave
-EXPOSE 3000
-CMD /usr/local/bin/helpwave
+FROM golang:1.19 AS builder
+
+WORKDIR /app
+COPY . /app
+
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o helpwave .
+
+
+FROM alpine AS production
+
+LABEL maintainer="development@helpwave.de"
+
+WORKDIR /app
+
+COPY --from=builder /app/helpwave .
+RUN chmod +x helpwave
+
+RUN touch .env
+
+ENV PORT 80
+ENV GIN_MODE release
+ENV LOG_LEVEL info
+ENV POSTGRES_PORT 5432
```

Comment:
Why set POSTGRES_PORT in the container, but everything else in the docker-compose? I don't think setup specific variables should be set in the container.

Reply:
Becuase `5432` is the default port set for postgresql. Keep in mind that you can still overwrite this later on using the `environment`-option in `docker-compose` (for example).